docs/pages/management/operations/scaling.mdx

```diff
@@ -42,6 +42,19 @@ teleport:
     max_users: 1000
 ```
 
+## Agent Configuration
+
+Consider bumping the default backoff parameters for agent caches. By default agents are fairly aggressive in trying
+to re-initialize their caches if they lose connectivity to the control plane. In very large clusters, this can contribute
```

Comment:
We don't use the "control plane" term in the docs. Does this mean "Auth and Proxy Services"?